Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9.10-slim
```

Comment:
Because it's the first version that works on ARM, was my reason, so I could test that the image would build on my machine, but again, we're going to grab the Dockerfile in Florentin's PR in the end, right